### ISSUE TITLE
empty/default sgdb doesn't pass terraform plan without errors

### ISSUE THREAD

**Issue by keen99** [dtan4/terraforming#115]
```
dbsg%% terraform plan

There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * aws_db_security_group.default: "ingress": required field is not set
```

dbsg.tf:

```
resource "aws_db_security_group" "default" {
    name        = "default"
    description = "default"

}
```

dbsg.tfstate:

```
{
  "version": 1,
  "serial": 1,
  "modules": [
    {
      "path": [
        "root"
      ],
      "outputs": {
      },
      "resources": {
        "aws_db_security_group.default": {
          "type": "aws_db_security_group",
          "primary": {
            "id": "default",
            "attributes": {
              "db_subnet_group_name": "default",
              "id": "default",
              "ingress.#": "0",
              "name": "default"
            }
          }
        }
      }
    }
  ]
}
```

**Comment by dtan4** [comment]
I fixed this bug at #120 to exclude DB security group with empty ingress rules.
This fix is released as v0.2.0 :shipit: 

If you still have something wrong, please reopen this issue.

**Comment by keen99** [comment]
:+1:   worked for me running master the other day, thanks!
